l10n_be_cooperator_portal/controllers/main.py

```diff
@@ -12,15 +12,18 @@
 
 
 class PortalTaxShelter(CustomerPortal):
```

Comment:
this should extend `CooperatorPortal` from the `cooperator_portal` module (which should be added to the dependencies), otherwise it removes its features.